pytorch_lightning/core/saving.py

```diff
@@ -383,6 +384,9 @@ def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
     # drop paramaters which contain some strange datatypes as fsspec
     for k, v in hparams.items():
         try:
+            if isinstance(v, Enum):
```

Comment:
This is like an option from the enum instance. Moreover, what I've been trying to do is make it as close to how omegaconf is saving it. So it can be compatible. Should I save it as a dict?

Reply:
Please recommend on my latest comment. Thanks!

Reply:
@omry Any guidance ?